wix-formbuilder/$w/WixForms/guides/Migrate from WixForms.md

```diff
@@ -0,0 +1,38 @@
+# Migrate from WixForms to WixFormsV2
+As of April 2025, [WixForms](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/introduction) is deprecated and any new forms created will only work with [WixFormsV2](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/introduction). You can access the functionality of WixForms using [WixFormsV2](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/introduction).
+
+Old forms will only work with the old APIs, WixForms, WixFormSubmitEvent, WixFormSubmittedErrorEvent, and WixFormSubmittedEvent. You can see which forms are old forms and which are new forms by looking at the forms and submissions page in a site dashboard, there's a tab for each. There is no easy way to distinguish in the editor, except that their menus are different.
+
+Below is a list of the deprecated WixForms components and functions and their alternatives in WixFormsV2.
+
+### [`collapsed`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/collapsed)
+Instead of `collapsed`, use [`collapsed`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/collapsed) from WixFormsV2.
```

Comment:
```suggestion
Instead of `collapsed` from `WixForms`, use [`collapsed`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/collapsed) from `WixFormsV2`.
```
I think being explicit about the fact that the deprecated one is from `WixForms` is especially important when the equivalent property in V2 has the same name -- hence the suggestion to add "from `WixForms`". If already making this change for those with identical names, I think we may as well word this way throughout.